### PR TITLE
docs: fix contributing md error

### PR DESCRIPTION
**Pull request by ivc369**
**What this PR does / why we need it**:

It should be `git remote add`, not `git remove add` 😀

**Which issue(s) this PR is related to** *(optional, link to 3rd issue(s))*:

Fixes #

Reference to #

**Special notes for your reviewer**:

/cc @your-reviewer

**Release note**:

```release-note
NONE
```

### CODE REVIEW

**Comment by zhujian7** [comment]
LGTM

File: web/src/components/stageTemplate/add/Input.js [hunk 1]
```diff
@@ -50,11 +50,11 @@ const InputSection = props => {
                             component={
                               a.name === 'cmd' ? TextareaField : InputField
                             }
-                            {...a.name === 'cmd' && {
+                            {...(a.name === 'cmd' && {
```
Comment (supereagle) [inline web/src/components/stageTemplate/add/Input.js]: This has been included in #1153, please rebase.

Comment (ivc369) [reply web/src/components/stageTemplate/add/Input.js]: Already rebased.

**Pull request merged**